aws/resource_aws_prometheus_workspace_test.go

```diff
@@ -0,0 +1,121 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/prometheusservice"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccAWSAMPWorkspace_basic(t *testing.T) {
+	workspaceAlias := acctest.RandomWithPrefix("tf_amp_workspace")
+	resourceName := "aws_prometheus_workspace.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSAMPWorkspaceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAWSAMPWorkspaceConfigWithAlias(workspaceAlias),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAWSAMPWorkspaceExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "alias", workspaceAlias),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAWSAMPWorkspaceConfigWithoutAlias(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "alias", ""),
+				),
+			},
+			{
+				Config: testAWSAMPWorkspaceConfigWithAlias(workspaceAlias),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAWSAMPWorkspaceExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "alias", workspaceAlias),
+				),
+			},
+		},
+	})
+}
+
```

Comment:
New resources should include disappears testing as noted in the [Contributing Guide section](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/running-and-writing-acceptance-tests.md#disappears-acceptance-tests):

```suggestion

func TestAccAWSAMPWorkspace_disappears(t *testing.T) {
	resourceName := "aws_prometheus_workspace.test"

	resource.ParallelTest(t, resource.TestCase{
		PreCheck:     func() { testAccPreCheck(t) },
		Providers:    testAccProviders,
		CheckDestroy: testAccCheckAWSAMPWorkspaceDestroy,
		Steps: []resource.TestStep{
			{
				Config: testAWSAMPWorkspaceConfigWithoutAlias(),
				Check: resource.ComposeTestCheckFunc(
					testCheckAWSAMPWorkspaceExists(resourceName),
					testAccCheckResourceDisappears(testAccProvider, resourceAwsPrometheusWorkspace(), resourceName),
				),
				ExpectNonEmptyPlan: true,
			},
		},
	})
}

```

Reply:
added.